### PR TITLE
[7.3] MethodChainingIndentationFixer - add tests for PHP 7.3

### PR DESCRIPTION
**Pull request by kubawerlos**

### CODE REVIEW

File: tests/Fixtures/Integration/misc/PHP7_3.test [hunk 1]
```diff
@@ -81,6 +82,7 @@ dirname(__FILE__, ); // `dir_constant` rule
 is_null($var, ); // `is_null` rule
 strpos($a, $b, ); // `mb_str_functions` rule
 sample('foo',    'foobarbaz', 'baz'   , ); // `method_argument_space` rule
+$user->setEmail('voff.web@gmail.com', ); // `method_chaining_indentation` rule
```
Comment (keradus) [inline tests/Fixtures/Integration/misc/PHP7_3.test]:
```
$user->setEmail('voff.web@gmail.com', ) // `method_chaining_indentation` rule
               ->setPassword('233434', );
```

now, there is nothing you are chaining here, so rule has nothing to do

Comment (kubawerlos) [reply tests/Fixtures/Integration/misc/PHP7_3.test]: fixed

**Comment by keradus** [comment]
Travis is failing on 7.3

**Comment by kubawerlos** [comment]
Fixed, `multiline_whitespace_before_semicolons` update affected integration test.

**Comment by keradus** [comment]
Thank you @kubawerlos.

**Pull request merged**